server/index.js

```diff
@@ -214,4 +223,48 @@ api.get('/torrents/:infoHash/archive', findTorrent, function (req, res) {
   archive.finalize();
 });
 
+function parseDom(body) {
+  var parseHandler = new htmlparser.DefaultHandler(function (err, dom) {
+    if (err) {
+      console.error("Error: " + err);
+    }
+  });
+  var parser = new htmlparser.Parser(parseHandler);
+  parser.parseComplete(body);
+  return parseHandler.dom;
+}
+
+api.get('/search/:param', function (req, res) {
+  request(`https://1337x.to/search/${encodeURIComponent(req.params.param)}/1/`, function(error, response, body) {
+    var dom = parseDom(body);
+    var rows = select(dom, '.search-page table tbody tr');
```

Comment:
I don't like the idea of scraping other websites, or even depending on any third parties, so I'm afraid I can't approve this